.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,43 @@
+---
+name: Bug report
+about: Create a report to help us improve
+
+---
+
+# Describe the bug #
+
+A clear and concise description of what the bug is.
+
+# Environment details #
+
+- Image/version of ceph-csi-operator :
+- Image/version of Ceph-CSI driver :
+- Helm chart version :
```

Comment:
This assumes a helm chart which is not yet part of the project. Should we add this after we introduce helm charts?

Reply:
Sure